www/src/pages/index.js

```diff
@@ -207,7 +207,7 @@ class IndexRoute extends React.Component {
                           css={{ marginBottom: rhythm(2) }}
                         />
                       ))}
-                      <CtaButton to="/blog/">Read More</CtaButton>
+                      <CtaButton to="/blog/" overrideCSS={{ marginBottom: rhythm(2) }}>Read More</CtaButton>
```

Comment:
Thank you for fixing this!

Could you please use `blockMarginBottom` instead of `rhythm` to define `marginBottom` (similar to this: https://github.com/gatsbyjs/gatsby/blob/c08b825c81c8d686a2b89a9ce0e5cb74c1043a26/www/src/pages/blog/index.js#L24)?

We're (unfortunately not consistently :-/ ;-)) trying to use `blockMarginBottom` for vertical margin/padding.

Reply:
```javascript
<CtaButton to="/blog/" overrideCSS={{ marginBottom: options.blockMarginBottom * 2 }}>Read More</CtaButton>
```

Like this?

Reply:
Close — you need to wrap the `options.blockMarginBottom * 2` with some `rhythm` ;-) …like this:

```
<CtaButton to="/blog/" overrideCSS={{ marginBottom: rhythm(options.blockMarginBottom * 2) }}>Read More</CtaButton>
```